docs/install_guides/installing-openvino-yocto.md

```diff
@@ -43,26 +43,26 @@ MACHINE = "intel-skylake-64"
 # Enable clDNN GPU plugin when needed.
 # This requires meta-clang and meta-oe layers to be included in bblayers.conf
 # and is not enabled by default.
-PACKAGECONFIG_append_pn-openvino-inference-engine = " opencl"
+PACKAGECONFIG:append:pn-openvino-inference-engine = " opencl"
 
 # Enable building inference engine python API.
 # This requires meta-python layer to be included in bblayers.conf.
-PACKAGECONFIG_append_pn-openvino-inference-engine = " python3"
+PACKAGECONFIG:append:pn-openvino-inference-engine = " python3"
 
-# This adds inference engine related libraries in the target image.
-CORE_IMAGE_EXTRA_INSTALL_append = " openvino-inference-engine"
+# This adds OpenVINO related libraries in the target image.
```

Comment:
```suggestion
# This adds Inference Engine related libraries in the target image.
```

Reply:
Inference Engine is OK because it's 2021.4.

Reply:
Thank you for reviewing. I have updated the PR.